plugins/woocommerce-admin/client/launch-your-store/hub/main-content/pages/launch-store-success/WhatsNext.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+import { Button } from '@wordpress/components';
+import { find } from 'lodash';
+import { useMemo } from '@wordpress/element';
+import type { TaskListType } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { ADMIN_URL } from '~/utils/admin-settings';
+
+type WhatsNextProps = {
+	activePlugins: string[];
+	allTasklists: TaskListType[];
+};
+
+type Action = {
+	title: string;
+	description: string;
+	link: string;
+	linkText: string;
+	trackEvent: string;
+};
+
+const getActionsList = ( { activePlugins, allTasklists }: WhatsNextProps ) => {
+	const actions: Action[] = [];
+	const pick = ( action: Action, condition: boolean ) => {
+		if ( actions.length < 3 && condition ) {
+			actions.push( action );
+		}
+	};
+
+	const isMailChimpActivated =
+		activePlugins.indexOf( 'mailchimp-for-woocommerce' ) !== -1;
+	const setupTasks = find( allTasklists, { id: 'setup' } )?.tasks || [];
+	const extendedTasks = find( allTasklists, { id: 'extended' } )?.tasks || [];
+
+	const isMarketingTaskCompleted =
+		find( setupTasks, { id: 'marketing' } )?.isComplete || false;
+
+	const isPaymentsTaskCompleted =
+		find( setupTasks, { id: 'payments' } )?.isComplete || false;
+
+	const isMobileTaskCompleted =
+		find( extendedTasks, { id: 'get-mobile-app' } )?.isComplete || false;
```

Comment:
Since you are removing Lodash, combining these three in a single `Array.reduce` would be lovely